joomla/joomla-cms
# [4.0] Generate an autoload_psr4.php with extensions namespaces

### Summary of Changes

Generates an autoload_psr4.php file with namespace and folder for the namespaced extensions. (First version, there needs to be done some follow ups)

### Testing Instructions

Apply patch, and reload the page. Make sure there are no errors or notices and than check if there in the libraries folder there is a 'autoload_psr4.php' file created. (but only on first load)

This should be also updated on extension update, install, publish / unpublish etc.

As when the plugin is not enabled this can break the entire site, we should reevaluate protecting plugins / extensions OR move it to the installer etc. 

### Expected result

### Actual result

### Documentation Changes Required

Probably
